### DISCUSSION TITLE
Allow me to hide people from my "For you" feed without blocking them

### DISCUSSION THREAD

**Discussion by fedecalendino** [community/community#19974] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi!

I wanted to say that I'm quite liking the "For you" section, but I feel that GitHub needs to give more control over it.

Just because I used to work/collaborated with someone in the past doesn't mean that I'm interested on which repos they star/follow/etc.

For example, I have this one ex-coworker (**that I don't follow**) that seems to mass-star repositories and what ends up happening is that my "For you" feed feels more like "About username". 

It would be nice to be able to "hide" the actions of certain users without having straight up blocking them.

![foryou](https://user-images.githubusercontent.com/7383708/177148730-dbd0c421-582b-4221-b261-d1e9d563bc1a.png)

**Comment by diegoleondev** [comment]
I think the same thing

**Comment by pcjmfranken** [comment]
Similarly, I'd like to have my own account completely excluded from directly appearing in or used to influence the feeds of others but for my profile to remain public. A complete "Feed" opt-out.

**Comment by mdsumner** [comment]
please can we have this? it'd be different if the For You suggestions were actually useful, but they are rubbish. You have the data, why suggest people I have followed in the past but don't follow now? why do you think that is? I don't want to see these horrible people

**Comment by siminapasat** [comment, marked as answer]
We're working on "user disinterest" which should address this feedback ☝️

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
